README.md

```diff
@@ -330,6 +350,118 @@ The `build_rpm.sh` script is designed to create LDMS producer RPM packages. It a
 
+---
+
+# **Building iDRAC Telemetry Containers**
+
+The build script now supports building containers for iDRAC Telemetry Reference Tools components. The script automatically clones the [iDRAC-Telemetry-Reference-Tools repository](https://github.com/dell/iDRAC-Telemetry-Reference-Tools) at commit `e86fecb` and uses its Dockerfiles to build the images.
+
+## Components
+
+### 1. **KafkaPump** (`kafkapump`)
+Data pump that publishes iDRAC telemetry data to Apache Kafka topics.
+
+### 2. **VictoriaPump** (`victoriapump`)  
+Data pump that publishes iDRAC telemetry data to VictoriaMetrics.
+
+### 3. **Telemetry Receiver** (`telemetry-receiver`)
+Complete telemetry collection service that includes:
+- `idrac-telemetry-receiver` - Main telemetry receiver
+- `dbdiscauth` - Database discovery and authentication
+- `configui` - Configuration UI  
+- `redfishread` - Redfish API reader
+
+## Usage Examples
+
+### Build All Telemetry Containers
+
+```bash
+# Build all three telemetry containers with default settings
+./build_images.sh telemetry
+
+# Build all with specific tag
+./build_images.sh telemetry image_tag=v1.0
+```
+
+### Build Individual Telemetry Containers
+
+```bash
+# Build only KafkaPump
+./build_images.sh kafkapump
+
+# Build only VictoriaPump with specific tag
+./build_images.sh victoriapump victoriapump_tag=v1.0
+
+# Build only Telemetry Receiver
+./build_images.sh telemetry-receiver telemetry_receiver_tag=latest
+```
+
+### Build and Push to Registry
+
+```bash
+# Push all telemetry containers to registry
+./build_images.sh telemetry \
+  build_tool=docker \
+  build_action=push \
+  image_tag=1.0.0
+
+# Push specific telemetry container
+./build_images.sh kafkapump \
+  build_tool=docker \
+  build_action=push \
+  kafkapump_tag=v1.0.0
+```
+
+### Build with Individual Tags
+
+```bash
+# Build all telemetry containers with different tags
+./build_images.sh telemetry \
+  kafkapump_tag=kafka-v1.0 \
+  victoriapump_tag=victoria-v1.0 \
+  telemetry_receiver_tag=receiver-v1.0
+
+# Build multiple with specific tags
+./build_images.sh kafkapump,victoriapump image_tag=1.0
+```
+
+## Container Details
+
+### Dockerfile Source
+
+Dockerfiles are sourced from the iDRAC-Telemetry-Reference-Tools repository:
+- **KafkaPump**: Uses `docker-compose-files/Dockerfile` with `CMD=kafkapump`
+- **VictoriaPump**: Uses `docker-compose-files/Dockerfile` with `CMD=victoriapump`
+- **Telemetry Receiver**: Uses `docker-compose-files/Dockerfile.telemetry_receiver`
+
+### Build Process
+
+1. Clones iDRAC-Telemetry-Reference-Tools repository at commit `e86fecb` (if not already cloned)
+2. Uses Dockerfiles from the cloned repository
+3. Builds static binaries from source code
+4. Creates minimal scratch-based images
+5. Includes SBOM and provenance metadata (when pushed to registry)
+
+### Image Characteristics
```

Comment:
do we need this Image Characteristics and parameters specific to telemetry, but its not there for others. can we remove it?